cetmix_tower_server/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Cetmix Tower Server Management",
     "summary": "Flexible Server Management directly from Odoo",
-    "version": "14.0.0.3.28",
+    "version": "14.0.0.3.29",
```

Comment:
Please set version to "14.0.0.4.0" and update the `migrations/` directory naming accordingly